jsonrpc/src/methods/history.json

```diff
@@ -287,13 +287,13 @@
   },
   {
     "name": "portal_historyOffer",
-    "summary": "Request message to offer a set of content_keys that this node has content available for.",
+    "summary": "Send the provided content item to interested peers.",
```

Comment:
Should it be send to _all_ interesting peers? Or can a subset be selected? Might want to clarify that in the description.

Reply:
Yeah, I dropped the word *all* from the summary, leaving that as a client implementation detail. For us, it's simplest to send to `log2(interested_peers)`, the same as we do whenever we find new content. I can add a line specifically saying that it's up to the client. I can imagine us wanting to actually send it to all peers when a node is acting as a bridge.